Source/x11/XGServerWindow.m

```diff
@@ -4409,75 +4415,140 @@ - (void) freecursor: (void*) cid
   return depth;
 }
 
+/* This method assumes that we deal with one X11 screen - `defScreen`.
+   Basically it means that we have DISPLAY variable set to `:0.0`.
+   AppKit and X11 use the same term "screen" with different meaning:
+   AppKit Screen - single monitor/display device.
+   X11 Screen - it's a plane where X Server can draw on.
+   XRandR - Xorg extension that helps to manipulate monitor layout providing
+   X11 Screen. 
+   We map XRandR monitors (outputs) to NSScreen. */
 - (NSArray *)screenList
 {
-  /* I guess screen numbers are in order starting from zero, but we
```

Comment:
We still need to put the default monitor first, the gui code relies on that.

Reply:
Done.